src/main/menu.ts

```diff
@@ -157,6 +157,14 @@ export default class MenuBuilder {
         },
         enabled: false,
       },
+      {
+        id: 'export',
+        label: 'Export Project',
+        accelerator: 'CommandOrControl+Shift+E',
```

Comment:
I believe (unless this has changed) the key combo suggested was CommandOrControl+E for EDL export, and CommandOrControl+Shift+E for video export.

Reply:
Not sure where this was mentioned

Reply:
My bad just checked figma. Updated :))